SharedPackages/BrowserServicesKit/Sources/Subscription/API/SubscriptionAPIService.swift

```diff
@@ -120,9 +123,9 @@ public struct DefaultSubscriptionAPIService: SubscriptionAPIService {
         let url = baseURL.appendingPathComponent(endpoint)
         var request = URLRequest(url: url)
         request.httpMethod = method
-        if let headers = headers {
-            request.allHTTPHeaderFields = headers
-        }
+        request.allHTTPHeaderFields = headers ?? [:]
+        request.allHTTPHeaderFields?[HTTPHeaderKey.userAgent] = userAgent
```

Comment:
Can the `headers ` already contain a user agent? This would override it

Reply:
No, this is only used by Subscriptions V1 and soon should be removed.